src/LibraryManager.Vsix/Commands/ManageLibrariesCommand.cs

```diff
@@ -56,7 +56,7 @@ private void BeforeQueryStatus(object sender, EventArgs e)
 
         private async Task ExecuteAsync(object sender, EventArgs e)
         {
-            Telemetry.TrackUserTask("ManageLibraries");
+            Telemetry.TrackUserTask("ManageLibrariesCommand");
```

Comment:
This looks like an operation. Will you consider TrackOperation() ?  
And the name should be like "Run-ManageLibrariesCommand".

Reply:
This is a menu command invoked by the use. Should it be tracked as an operation?